client/components/App.jsx

```diff
@@ -10,6 +11,7 @@ const App = () => {
       <Switch>
         <Route path='/register' component={Register} />
         <Route path ='/' component={Header} />
+        <Route path='/progressbar' component={ProgressBar} />
         <h1>MOA development has begun!</h1>
       </Switch>
     </Router>
```

Comment:
My bad this shouldn't be called from app when you submit this PR that's just so you can check it renders. Please remove changes to app.